tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,159 @@
+- name: create dataplane-adoption-secret.yaml
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: dataplane-adoption-secret
+        namespace: openstack
+    data:
+        ssh-privatekey: |
+    $(cat {{ edpm_privatekey_path }} | base64 | sed 's/^/        /')
+    EOF
+
+- name: get node IP
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    virsh --connect=qemu:///system -q domifaddr director_standalone | awk 'NF>1{print $NF}' | cut -d/ -f1
+  register: node_ip_result
```

Comment:
I'm trying to capture the IP of the standalone box,
@marios will that work for CI? If not, how can get the standalone IP?

Reply:
no we'll have to add something to capture the IP and set some vars in the job itself

for example you can see in the log of the jobs that ran here https://logserver.rdoproject.org/31/31/cfa9a03c8297479cfdc759dd08c380f5c90bfb7a/github-check/data-plane-adoption-github-rdo-centos-8-crc-centos-9-standalone/4844fa2/zuul-info/inventory.yaml the standalone node is the 'primary' 


```
    primary:
      ansible_connection: ssh
      ansible_host: 38.102.83.51
      ansible_port: 22
      ansible_python_interpreter: auto
```

Reply:
eg something similar to this that was done for the edpm:

https://github.com/rdo-infra/rdo-jobs/blob/e341d12abd1b9cb3b5e964ba75b11402b57d22ff/playbooks/edpm/deploy_podified_control_plane.yaml#L37-L45

```
    - name: Get subnode IP
      register: subnode_ip
      ansible.builtin.slurp:
        path: "/etc/nodepool/sub_nodes_private"

    - name: Get Controller IP
      ansible.builtin.slurp:
        path: /etc/nodepool/node_private
      register: controller_ip

```

Reply:
so it will have to go into the rdo-jobs (for now) since that is where the job plays live

I will try and dig more into this but not until end of week as I'm ci ruck currently (and solo)

Reply:
I think the IP should be an input variable rather than trying to autodetect it. Different developer/CI environments might have different ways of obtaining the IP, we should not assume we're using Vagrant.

Anyway, we talked with @fao89 that we'd like to merge this and iterate in further PRs. It would make it easier to review new changes.